### ISSUE TITLE
Support expand query

### ISSUE THREAD

**Issue by eddiegroves** [MrRefactoring/confluence.js#4]
Hi there

Thanks for this tidy Confluence API library!

I've noticed that `expand` is supported by query parameters but is not actually passed through to the request query.

Example

```
const contents = await client.contentAttachments.getAttachments({
  id: "123123",
  filename: "test.png",
  expand: ["metadata.properties"]
});
```

Results in this request
```
GET https://somewhere.atlassian.net/wiki/rest/api/content/1595999911/child/attachment?filename=test.png
```

I was expecting

```
GET https://somewhere.atlassian.net/wiki/rest/api/content/1595999911/child/attachment?filename=test.png&expand=metadata.properties
```

Cheers!

**Comment by MrRefactoring** [comment]
Nice catch, will fix it soon. Thank you for contribution!

**Comment by MrRefactoring** [comment]
It was fixed in `v1.1.1`. Try something like this:

```ts
import { Parameters } from 'confluence.js';

const contents = await client.contentAttachments.getAttachments({
  id: '123123',
  filename: 'test.png',
  expand: [Parameters.GetAttachments.Expand.PropertiesMetadata],
});
```
